### ISSUE TITLE
Organize the menu structure

### ISSUE THREAD

**Issue by ctrueden** [fiji/fiji#167]
As discussed on the ImageJ forum in a couple of different threads ([1](http://forum.imagej.net/t/2442), [2](http://forum.imagej.net/t/5047), we should invest some effort again in making the Fiji menu structure coherent and consistent.

A good starting point would be @tferr's [incomplete list of "misplaced" menu entries](http://forum.imagej.net/t/customize-menus-in-fiji/2442/7?u=ctrueden), and maybe my [proposed changes to the Help menu](http://forum.imagej.net/t/connection-problems-with-imagej-nih-gov-ij-developer-macro-functions-html/5047/5?u=ctrueden).

I might be able to have a student at LOCI work on this over the summer; we'll see how things go.

**Comment by rcatwood** [comment]
Agree! Recently a colleague at University of Manchester explained his routine including a bilateral image filter, and I was puzzled because I had never seen that available in Imagej/Fiji, and I couldn't find it. But it is there, in a deep menu somewhere.

**Comment by imagejan** [comment]
@rcatwood
> But it is there, in a deep menu somewhere.

The [command finder](https://imagej.net/Using_the_Command_Launcher) is extremely helpful for that.
Hopefully, recent work by @ctrueden in https://github.com/imagej/imagej-legacy/compare/search-bar will make this functionality more visible to users.

**Comment by ctrueden** [comment]
Another topic discussing this and related issues is [here](https://forum.image.sc/t/would-it-be-better-to-list-plugins-by-feature-ontology/5124). I agree about the search bar making it easier to find things. But of course that is no excuse not to make the menus also nice to browse. :-)

**Comment by tischi** [comment]
@ctrueden Yes, this would be great! Also sounds like quite some work that could involve complex discussions. 

As a quick improvement that could maybe be done soon, could we just have another top level menu for all the plugins that are below the line (i.e. in the below screenshot under Volume Viewer):

<img width="235" alt="image" src="https://github.com/fiji/fiji/assets/2157566/ad2118cb-cf8d-4f10-8e95-d40a5b71b0e1">

I think those are SciJava style plugins?!

The reason I am asking for some quick fix is this issue: https://imagesc.zulipchat.com/#narrow/stream/327238-Fiji/topic/Cannot.20access.20all.20plugins

**Comment by imagejan** [comment]
See also https://github.com/imagej/imagej-legacy/issues/179 and https://github.com/imagej/imagej-legacy/issues/270.
